packages/flex-dev-utils/src/fs.ts

```diff
@@ -460,6 +460,19 @@ export const getPaths = () => {
       // public/*
       publicDir,
       appConfig: resolveRelative(publicDir, 'appConfig.js'),
+
+      // dependencies
+      dependencies: {
+        react: {
+          version: readPackageJson(resolveRelative(nodeModulesDir, 'react', 'package.json')).version,
```

Comment:
is it possible that dependency fail to resolve? Previous code was doing an assert https://github.com/twilio/flex-plugin-builder/blob/0a78ae0596752727d4eae6412c407e35e516075f/packages/flex-dev-utils/src/fs.ts#L132

Reply:
If it does, the script should fail to start - these are required packages and must be installed.